device/dell/x86_64-dell_s6000_s1220-r0/installer.conf

```diff
@@ -1,4 +1,5 @@
 ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="processor.max_cstate=1 intel_idle.max_cstate=0"
+DOCKER_IN_RAM=y
```

Comment:
for testing purpose only, will remove.

Reply:
You mentioned 'will remove', when? Is it in this PR?

---
In reply to: [310374306](https://github.com/Azure/sonic-buildimage/pull/3279#discussion_r310374306) [](ancestors = 310374306)